src/parser/jobs/sch/modules/ScholarCooldownDowntime.tsx

```diff
@@ -0,0 +1,23 @@
+import {Trans} from '@lingui/react'
+import {ActionLink} from 'components/ui/DbLink'
+import {CooldownDowntime} from 'parser/core/modules/CooldownDowntime'
+import React, {Fragment} from 'react'
+
+export default class ScholarCooldownDowntime extends CooldownDowntime {
+	override trackedCds = [
+		{
+			cooldowns: [this.data.actions.CHAIN_STRATAGEM],
+			allowedAverageDowntime: 7500,
+			firstUseOffset: 10000,
+		},
+		{
+			cooldowns: [this.data.actions.AETHERFLOW],
+			allowedAverageDowntime: 5000,
+			firstUseOffset: 10000,
```

Comment:
Same deal as Chain Strat here.

Reply:
These were the original numbers from the chainstrat module in the endwalker branch. I just copied them over.

Reply:
Doesn't look like they were updated for 6.0 then, will need to handle that. If the PR is purely to add Aetherflow, it should *only* add Aetherflow tho. If you're doing a more general update for Cooldown Downtime then the numbers should be fixed. Up to you which you prefer.